Sofascore/Sofascore/ViewControllers/EventDetailsViewController.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import SofaAcademic
+import UIKit
+import SnapKit
+
+class EventDetailsViewController: UIViewController {
+    
+    private let detailsLabel: UILabel = .init()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addViews()
+        setupConstraints()
+        styleViews()
+    }
+    
+    @objc private func dismissButtonTapped() {
+        self.dismiss(animated: true, completion: nil)
```

Comment:
```suggestion
        dismiss(animated: true, completion: nil)
```

Mislim da mozes i bez self